docs/changelog/1_0_rc2.rst

```diff
@@ -301,8 +301,9 @@ Server configuration
     - ``never``
 
 ``EDGEDB_SERVER_TLS_CERT_MODE``
-    - ``file == default``
-    - ``self_signed`` — generate self-signed on bootstrap and when expired
+    - ``require_file`` (``== default`` if EDGEDB_SERVER_SECURITY is ``strict``)
+    - ``generate_self_signed`` generate self-signed certificate and private
+      key on bootstrap if certificate or key are not specified or missing.
```

Comment:
You probably want this in rc3 changelog

Reply:
We had this in rc2 changelog :laughing:

Reply:
We probably want to delete this from rc2 and add to rc3?

Reply:
Yes.